pkg/minikube/cluster/commands.go

```diff
@@ -122,6 +122,10 @@ func GenLocalkubeStartCmd(kubernetesConfig KubernetesConfig) (string, error) {
 		flagVals = append(flagVals, "--apiserver-name="+kubernetesConfig.APIServerName)
 	}
 
+	if kubernetesConfig.DNSDomain != "" {
+		flagVals = append(flagVals, "--dns-domain="+kubernetesConfig.DNSDomain)
+	}
+
```

Comment:
Is there a default value?  If so we have to be careful just like the --apiserver-name flag